objects/obj_splash_manager/Draw_64.gml

```diff
@@ -0,0 +1,27 @@
+
+// Draw the background
+draw_sprite_stretched(spr_color_light_blue, 0, 0, 0, room_width, room_height)
+
+draw_set_font(font_press_start_sm)
+draw_set_color(#0F68EF)
+
+var center_x = room_width/2
+
+draw_sprite(spr_logo_medrecruit_blue, 0, center_x, 340)
+
+draw_set_halign(fa_center) // center text
+
+draw_set_font(font_press_start_lg)
+draw_text(center_x, 520, "MEDRACER")
```

Comment:
we should rename the project in setting so the window etc. show MedRacer as the name

Reply:
Will do!